stories/Containers/LeftSideBar.stories.tsx

```diff
@@ -0,0 +1,307 @@
+                        field: 'QR code',
+                        isRequired: false,
+                        isRecommended: true
+                    }
+                }
+            },
+            settingsElements: {
+                key: 'Settings',
+                editorCategory: 'Settings',
+                draggableComponents: {
+                    DATE: {
+                        key: '26',
+                        field: 'Date',
+                        isRequired: true,
+                        isRecommended: false
+                    },
+                    TIME: {
+                        key: '27',
+                        field: 'Time',
+                        isRequired: true,
+                        isRecommended: false
+                    }
+                }
+            }
+        },
+        ElementWithCategory: [
+            {
+                editorCategory: 'Text',
+                field: [
+                    'Name of business',
+                    'Address of business',
+                    'Business contact information',
+                    'Name of sales associate',
+                    'Name of product or service',
+                    'Order type',
+                    'Payment method'
+                ]
+            },
+            {
+                editorCategory: 'Image',
+                field: [
+                    'Business logo',
+                    'Additional branding'
+                ]
+            },
+            {
+                editorCategory: 'Layout',
+                field: [
+                    'Menu table',
+                    'White space',
+                    'Divider'
+                ]
+            },
+            {
+                editorCategory: 'Number',
+                field: [
+                    'Station number',
+                    'Quantity of product or service',
+                    'Number of guests',
+                    'Transaction order number',
+                ]
+            },
+            {
+                editorCategory: 'Price',
+                field: [
+                    'Sale price',
+                    'Total amount of tax',
+                    'Rate of sales tax',
+                    'Total price of sale',
+                    'Tips',
+                    'Total price of sale with tax',
+                    'Total price with tip'
+                ]
+            },
+            {
+                editorCategory: 'Codes',
+                field: [
+                    'UPC code',
+                    'QR code'
+                ]
+            },
+            {
+                editorCategory: 'Settings',
+                field: [
+                    'Date',
+                    'Time'
+                ]
+            }
+        ]
+    }
+} as Meta;
+
+export const Basic: Story<LeftSideBarProps> = (args) => (
+    <LeftSideBar 
+        {...args}
+        onDragEnd={() => console.log('I have been dragged!')}  
```

Comment:
This should be in teh argTypes so the Actions tab Registers the Event. Look at the other files